builder/index.go

```diff
@@ -75,7 +75,7 @@ func parseIndexTemplate() (*template.Template, error) {
 
```

Comment:
the whole html thing can be removed

Reply:
I wouldn't do it - the functionality of the local relay is still there